runtime/common/src/lib.rs

```diff
@@ -16,10 +16,7 @@ pub type Nonce = u32;
 /// up by `pallet_aura` to implement `fn slot_duration()`.
 ///
 /// Change this to adjust the block time.
-#[cfg(not(feature = "paseo"))]
 pub const MILLISECS_PER_BLOCK: u64 = 6000;
-#[cfg(feature = "paseo")]
-pub const MILLISECS_PER_BLOCK: u64 = 12000;
```

Comment:
Should we be removing 12 second block times already? I would like to see a structured strategy for the next Pop Net release for enabling async backing that we could thoroughly test.

Reply:
Note: I would love to see async backing enabled on on our testnet :)

Reply:
reverted enablement